plugins/by-name/git-worktree/default.nix

```diff
@@ -59,12 +59,11 @@ lib.nixvim.plugins.mkNeovimPlugin {
   extraConfig =
     cfg:
     lib.mkIf cfg.enable {
-      assertions = [
-        {
-          assertion = cfg.enableTelescope -> config.plugins.telescope.enable;
-          message = ''Nixvim: The git-worktree telescope integration needs telescope to function as intended'';
-        }
-      ];
+      assertions = lib.nixvim.mkAssertions "plugins.git-worktree" {
+        assertion = cfg.enableTelescope -> config.plugins.telescope.enable;
+
+        message = "The git-worktree telescope integration needs telescope to function as intended.";
```

Comment:
These telescope messages should probably be rewritten, too. Feels redundant with prefixing the plugin name.